pablomt/pablomt-MexicoCity.md

```diff
@@ -3,3 +3,6 @@
 ### Restaurants
 
 ### Things to do
+
```

Comment:
What about museums,  Mexico City has great places to visit?
I also like La Condesa and Coyoacán